dbms/src/Server/StorageConfigParser.cpp

```diff
@@ -643,7 +643,7 @@ void StorageRemoteCacheConfig::parse(const String & content, const LoggerPtr & l
     readConfig(table, "dtfile_level", dtfile_level);
     RUNTIME_CHECK(dtfile_level <= 100);
     readConfig(table, "delta_rate", delta_rate);
-    RUNTIME_CHECK(std::isgreaterequal(delta_rate, 0.1) && std::islessequal(delta_rate, 1.0), delta_rate);
+    RUNTIME_CHECK(std::isgreaterequal(delta_rate, 0.0) && std::islessequal(delta_rate, 1.0), delta_rate);
```

Comment:
Yes, this change is to allow setting to 0.0 in order to bypass the Delta Cache checking.

However, currently the Delta Cache works as "no limit" when limit is set to 0. It is another story though.

Reply:
Why we need "no limit" to bypass the Delta Cache checking?

Reply:
According to metrics, occupy space 80% usually takes ~500ms for now. Allowing unlimited page cache help us benchmark the performance without occupy space.

Reply:
As `0.0` is tends to allow for performance testing. But it will lead to disk full is prod-env. Maybe add a warning log and some comments when it is set to `0.0`.